Assignment3.java

```diff
@@ -3,13 +3,24 @@
 public class Assignment3 {
```

Comment:
Yeah it is, but I just name all the assignments with their numbers, I made it sort of a convention :D